Makefile

```diff
@@ -6,9 +6,18 @@ VERSION := $(shell git describe --tags)
 COMMIT := $(shell git rev-parse HEAD)
 LEDGER_ENABLED ?= true
 DOCKER := $(shell which docker)
+DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf
 IMAGE_TAG = likecoin/likecoin-chain:$(VERSION)
 RBUILDER_IMAGE_TAG = cf0d1a9f3731e30540bbfa36a36d13e4dcccf5eb
 BUILDDIR ?= $(CURDIR)/build
+GOPATH ?= '$(HOME)/go'
+GOLANG_VERSION        ?= 1.17.2
+GOLANG_CROSS_VERSION  := v$(GOLANG_VERSION)
+GOGO_PROTO_URL      = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
+COSMOS_SDK_URL      = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.43.0
+COSMOS_PROTO_URL    = https://raw.githubusercontent.com/regen-network/cosmos-proto/master
+GOGO_PROTO_TYPES    = proto/gogoproto
+COSMOS_PROTO_TYPES  = proto/cosmos_proto
```

Comment:
I think it is to replace gen_proto.sh?

Than we should remove the gen_proto.sh file.